Sources/ComposableArchitecture/Effects/Cancellation.swift

```diff
@@ -101,7 +102,7 @@ extension Effect {
   /// - Parameter id: An effect identifier.
   /// - Returns: A new effect that will cancel any currently in-flight effect with the given
   ///   identifier.
-  public static func cancel<ID: Hashable>(id: ID) -> Self {
+  public static func cancel(id: some Hashable & Sendable) -> Self {
```

Comment:
Cancellation IDs must now be `Sendable`. Technically we can avoid this with:

```swift
@autoclosure @escaping @Sendable () -> some Mashable
```

But I'm not sure it's worth it.